otelcol/configprovider.go

```diff
@@ -14,6 +14,16 @@ import (
 	"go.opentelemetry.io/collector/confmap/provider/httpprovider"
 	"go.opentelemetry.io/collector/confmap/provider/httpsprovider"
 	"go.opentelemetry.io/collector/confmap/provider/yamlprovider"
+	"go.opentelemetry.io/collector/featuregate"
+)
+
+// PreventEnvVarExpansionFeatureGate is the feature gate that controls whether the collector
+// supports configuring the OpenTelemetry SDK via configuration
```

Comment:
```suggestion
// PreventEnvVarExpansionFeatureGate disables the ability to use environment variables
// without wrapping the variable in braces.
```
I don't think we need to explicitly call out SDK configuration here; I also don't think this has to do with the Collector's SDK instance.

Reply:
Looks like I had a bad copy-paste